src/app.js

```diff
@@ -171,16 +171,17 @@ async function getVideoData () {
         error(`failed to parse the lesson page '${urlValue}'}`)
       }
     } else {
-      let lessonURLs = []
       success('The URL is a playlist or series')
 
       // get the urls of the lessons
-      const re = /<a class="flex h-100 absolute top-0 right-0 bottom-0 left-0" style="pointer-events:all;" href="(\/lessons\/.+?)"/g
+      const re = /<a .* href="(\/lessons\/.+?)"/g
       // regexp in js have no matchAll method or something similar..
       let match
+      const lessonURLsSet = new Set()
       while ((match = re.exec(source))) {
```

Comment:
I think this not concerned with "Fix finding lesson url"

Reply:
I used set because with new lessons regexp there were duplication matches